src/components/Icon/IconArrow.test.js

```diff
@@ -0,0 +1,10 @@
+/* global describe, expect, it, shallow */
+import React from 'react';
+import IconArrow from './IconArrow';
+
+describe('IconArrow', () => {
+    it('has the appropriate classes', () => {
```

Comment:
Can we have a test that validates each icon renders SVG? I don't think we need to verify much (if any) of the SVG content but it would be useful to have a test to prove the component actually renders.